Assets/Scripts/DebugPlayerPrefsDeleter.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+public class DebugPlayerPrefsDeleter : MonoBehaviour
+{
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.A))
+        {
+            PlayerPrefs.DeleteAll();
```

Comment:
I think it would be good to log that PlayerPrefs were deleted here in case someone working on this project presses the A key and doesn't know about this script.